content/docs/introduction/getting_started.md

```diff
@@ -82,10 +78,9 @@ The number of OS threads executed by Prometheus is controlled by the
 the number of cores available.
 
 Blindly setting `GOMAXPROCS` to a high value can be
-counterproductive. See the relevant [Go
-FAQs](http://golang.org/doc/faq#Why_no_multi_CPU).
+counterproductive. See the relevant [Go FAQs](http://golang.org/doc/faq#Why_no_multi_CPU).
 
-Note that Prometheus by default uses around 3GB in memory. If you have a
+NOTE: **Note:** Prometheus by default uses around 3GB in memory. If you have a
```

Comment:
This stutters. Given the colloquial (and overly detailed) nature of the rest of this section, I don't think this needs to be emphasised.